src/tests/system/tests/test_identity.py

```diff
@@ -496,3 +496,42 @@ def test_identity__lookup_users_fully_qualified_name_and_case_insensitive(client
         result = client.tools.id(name)
         assert result is not None, f"User {name} was not found using id"
         assert result.memberof([103, 1003]), f"User {name} is member of wrong groups"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.authentication
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+def test_identity__lookup_user_posix_and_non_posix(client: Client, provider: GenericADProvider):
+    """
+    :title: with ldap provider idmapping is disabled
+    :setup:
+        1. Configure ldap_provider with idmapping disabled
```

Comment:
The setup steps should follow the code, and generally end right before the assertion. 
1. Create user with POSIX attributes
2. Create group with POSIX attributes
3. Create user with no POSIX attributes
4. Create group with no POSIX attributes
5. Configure SSSD with "ldap_id_mapping" = false
6.  Start SSSD